.docs/content/1.concepts/9.mongodb.md

```diff
@@ -0,0 +1,41 @@
+# MongoDB client connection problem
+
+[MongoDB](https://www.mongodb.com/) is a document-oriented database used to build highly available and scalable applications. MongoDB solution is for high-volume data storage and uses collections and documents, unlike traditional databases. It has been noticed that when a client is connected to MongoDB, multiple connections are created. This document intends to explain why.
+
+## MongoDB in ArmoniK
+MongoDB is used in ArmoniK as database to store the metadata of the tasks. Each compute compute must connect to MongoDB database. We have in ArmoniK several connections to mongoDB per pod. The number of connections often varies between 3 and 4 or more per compute pod.  We also noticed multiple initializations of the client in ArmoniK but this will be investigated in another PR independently. 
```

Comment:
```suggestion
MongoDB is used in ArmoniK as database to store the metadata of the tasks, resuts, sessions and authentication. Each compute compute must connect to MongoDB database. We have in ArmoniK several connections to mongoDB per pod. The number of connections often varies between 3 and 4 or more per compute pod.  We also noticed multiple initializations of the client in ArmoniK but this will be investigated in another PR independently. 
```